adbackend/src/main/java/com/csc/adbackend/Campaign.java

```diff
@@ -67,6 +67,14 @@ public void setName(String name) {
         return new ArrayList<>(this.ads.keySet());
     }
 
+    public Map<Integer, Ad> getAds() {
```

Comment:
Adds extra field to JSON representation (I think because jackson uses the get/set methods to do this). Use listOfAds() instead.